anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/MonitoredDetector.java

```diff
@@ -46,11 +46,10 @@ public MonitoredDetector(AnomalyDetector detector) {
 
     @Override
     public MappedMpoint classify(MappedMpoint mappedMpoint) {
-        MappedMpoint mappedPoint = detector.classify(mappedMpoint);
         PerformanceMonitor perfMonitor = new PerformanceMonitor(new PerfMonHandler(), new RmseEvaluator(), MAX_TICKS);
```

Comment:
We don't want to create a new perfmon handler or RMSE evaluator on every classification. That could be thousands per second. Pass the perfmon into the monitored detector when you create that.